build/rollup.config.js

```diff
@@ -54,44 +52,41 @@ export default inputs.map(input => {
                 i18n: '__',
                 async: 'async',
                 handlebars: 'handlebars',
+                'idb-wrapper': 'IDBStore',
                 'lib/uuid': 'lib/uuid',
-                'lib/store/idbstore': 'lib/store/idbstore',
                 'lib/decimal/decimal': 'lib/decimal/decimal',
                 'lib/expr-eval/expr-eval': 'lib/expr-eval/expr-eval',
-                'lib/polyfill/es6-promise': 'lib/polyfill/es6-promise',
+                'webcrypto-shim': 'webcrypto-shim',
                 ...inputGlobalNames
             }
         },
         external: [
             ...localExternals,
-            ...libExternals,
-            'jquery',
-            'lodash',
+            'async',
+            'context',
             'handlebars',
-            'moment',
             'i18n',
-            'async',
+            'jquery',
             'jquery.fileDownload',
-            'module',
-            'context',
-            'lib/uuid',
-            'lib/store/idbstore',
             'lib/decimal/decimal',
-            'lib/expr-eval/expr-eval'
+            'lib/expr-eval/expr-eval',
+            'lib/uuid',
+            'lodash',
+            'module',
+            'moment'
         ],
         plugins: [
+            resolve(),
+            commonJS({
+                include: 'node_modules/**'
```

Comment:
Redundant `include` config, because this what `resolve()` does.

"Locate modules using the Node resolution algorithm, for using third party modules in `node_modules.`"